### PR TITLE
TSDB: add reindex time_series tests

### PR DESCRIPTION
**Pull request by weizijun**
add reindex time_series yaml tests

### CODE REVIEW

**Comment by csoulios** [comment]
@elasticmachine test this please

**Comment by weizijun** [comment]
@elasticmachine update branch

File: modules/reindex/src/yamlRestTest/resources/rest-api-spec/test/reindex/100_tsdb.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      version: " - 8.0.99"
+      reason: introduced in 8.1.0
+
+  - do:
+      indices.delete:
+        index: tsdb, standard, tsdb_new
+        ignore_unavailable: true
+
+---
+reindex tsdb index:
```
Comment (nik9000) [inline modules/reindex/src/yamlRestTest/resources/rest-api-spec/test/reindex/100_tsdb.yml:71]: It'd be cool to have a test reindexing from a TSDB index into a non-tsdb index too.

**Comment by nik9000** [comment]
@elasticmachine ok to test

File: modules/reindex/src/yamlRestTest/resources/rest-api-spec/test/reindex/100_tsdb.yml (hunk 1 above)
Comment (weizijun) [reply modules/reindex/src/yamlRestTest/resources/rest-api-spec/test/reindex/100_tsdb.yml:71]: the case fisrt reindex tsdb index to standard index, then reindex standard index to tsdb index

**Review by nik9000 — APPROVED** [review]

**Pull request merged**